src/views/shared/SecurityBeacon.js

```diff
@@ -15,6 +15,18 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
   var _ = Okta._,
       $ = Okta.$;
 
+  // TODO(OKTA-118980): Server should send description as part of appState.
+  function getSecurityImageDescriptionFromUrl(imgSrc) {
+    var parser = document.createElement('a');
+    parser.href = imgSrc;
+
+    // Get stuff between the last slash, and the first period after that slash.
+    // (if the string ends with a slash, use the second-to-last slash instead.)
+    var match = /^.*\/([^.]+).*\/?$/.exec(parser.pathname);
+    var basename = (match.length >= 2) ? match[1] : null;
+    return basename ? (basename + ' security image') : '';
```

Comment:
Thinking about it more, I think the API should return you this info... we can talk about it tomorrow. @hor-kanchan-okta also likes that idea better....